### PR TITLE
PA-2388 Updated Caddy Config

### PR DESCRIPTION
**Pull request by Fosol**
Caddy configuration doesn't handle if the URL has a path.  We'll need to update.

### CODE REVIEW

File: openshift/4.0/templates/maintenance/README.md [hunk 1]
```diff
@@ -68,21 +67,49 @@ You may need to manually run the build config.
...
 ```conf
-PROJECT_NAMESPACE=354028
-ROLE_NAME=proxy
 ENV_NAME=dev
+IMAGE_TAG=dev
 APP_DOMAIN=proxy-caddy-pims-dev.apps.silver.devops.gov.bc.ca
 APP_PORT=2015
+CPU_REQUEST=100m
+CPU_LIMIT=1
+MEMORY_REQUEST=500Mi
```
Comment (jdtoombs) [inline openshift/4.0/templates/maintenance/README.md:84]: Today I learned what a mebibyte is... lol

**Review by jdtoombs — APPROVED** [review]

File: openshift/4.0/templates/maintenance/README.md (hunk 1 above)
Comment (agalbrai) [reply openshift/4.0/templates/maintenance/README.md:84]: me too haha

**Review by devinleighsmith — APPROVED** [review]

**Comment by devinleighsmith** [comment]
Wait, so this doesn't fix the caddyfile issue?

**Comment by Fosol** [comment]
No it doesn't. I spent an hour trying a few different configurations but couldn't find one that would return the maintenance page no matter what the original path was.

**Pull request merged**